server/views/caseList/CATable.pug

```diff
@@ -0,0 +1,47 @@
+table#hdcEligiblePrisoners.largeMarginBottom
+    thead
+        tr
+            th Name
+            th NOMS ID
+            th Location
+            th HDCED
+            th CRD
+            th Current status
+            th
+    tbody
+        if hdcEligible && hdcEligible.length > 0
+            each prisoner in hdcEligible
+                tr.hdcEligible
+                    td.name
+                        if prisoner.lastName
+                            | #{prisoner.lastName}
+                            if prisoner.firstName
+                                | , #{prisoner.firstName}
+
+                    td.offenderNo
+                        if prisoner.offenderNo
+                            | #{prisoner.offenderNo}
+
+                    td.location
+                        if prisoner.internalLocationDesc
+                            | #{prisoner.internalLocationDesc}
+                        if prisoner.internalLocationDesc && prisoner.agencyLocationDesc
+                            =" - "
+                        if prisoner.agencyLocationDesc
```

Comment:
`if prisoner.agencyLocationDesc` is redundant because of `if prisoner.internalLocationDesc && prisoner.agencyLocationDesc` ?